lte/gateway/c/oai/tasks/mme_app/mme_app_sgsap_location_update.c

```diff
@@ -435,23 +435,7 @@ int mme_app_handle_nas_cs_domain_location_update_req(
   }
   mme_config_unlock(&mme_config);
 
-/*TODO CSFB, currently from HSS access_mode is rceived as PACKET_ONLY
- * For testing purpose we are commenting, later we will modify code as below
- */
-#if 0
-  if((ue_context->access_mode == NAM_PACKET_AND_CIRCUIT) &&
-    (ue_context->sgs_context->ts6_1_timer.id == MME_APP_TIMER_INACTIVE_ID)) {
-    /*Send SGSAP Location Update Request message to SGS task*/
-    send_itti_sgsap_location_update_req(ue_context);
-    OAILOG_DEBUG (LOG_MME_APP, "Sending Location Update message to SGS task");
-  }else if(ue_context->sgs_context->ts6_1_timer.id != MME_APP_TIMER_INACTIVE_ID) {
-    //Ignore the the messae as Location Update procedure is already triggered
-    OAILOG_DEBUG (LOG_MME_APP, "Dropping the message as Location Update procedure is already triggered for UE %d\n",
-    itti_nas_location_update_req->ue_id);
-  }
-
-#endif
-  if (/*(ue_context->access_mode == NAM_PACKET_AND_CIRCUIT) && */
+  if ((ue_context->network_access_mode == NAM_PACKET_AND_CIRCUIT) &&
```

Comment:
If it is PACKET_ONLY, are we sending response message for the received message?
Looks like there is no reply, could you please check it?

Reply:
If it is network access mode is PACKET_ONLY NAS is not sending cs_domain_location_update_req to mme_app, so no need to send response back to NAS.